libs/wingsdk/src/target-tf-azure/function.ts

```diff
@@ -152,11 +152,13 @@ export class Function extends cloud.FunctionBase {
         },
       },
       httpsOnly: true,
-      appSettings: {
-        ...this.env,
-        WEBSITE_RUN_FROM_PACKAGE: `https://${this.storageAccount.name}.blob.core.windows.net/${functionCodeBucket.storageContainer.name}/${functionCodeBlob.name}`,
-        FUNCTIONS_WORKER_RUNTIME: functionRuntime,
-      },
+      appSettings: Lazy.anyValue({
```

Comment:
Hopefully this isn't too much of a "gotcha" over time, having to know what values may be mutated by bindings in presynth.

Reply:
Yeah these issues are hard to deal with. The original Wing language proposal mentions an idea of how the language might be able to handle tokens and other lazy values more natively, which could avoid the need for code like this in userland perhaps. https://docs.winglang.io/contributors/rfcs/2022-05-28-winglang-reqs#opaque-primitives-tokens